cassandra/src/main/scala/filodb.cassandra/columnstore/CassandraColumnStore.scala

```diff
@@ -131,7 +132,8 @@ extends CachedMergingColumnStore with CassandraColumnStoreScanner with StrictLog
     val keyspace = clusterConnector.keySpaceName(dataset)
     require(splitsPerNode >= 1, s"Must specify at least 1 splits_per_node, got $splitsPerNode")
 
-    val tokensByReplica = metadata.getTokenRanges.asScala.toSeq.groupBy { tokenRange =>
+    val tokenRanges=unwrapTokenRanges(metadata.getTokenRanges.asScala.toSeq)
```

Comment:
Can we add spaces around =?

Also, converting to a java hashset and then converting back to scala seems inefficient.  Let me see if there is a better way to do this.